src/main/webapp/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button onclick="loadSchoolInfo()">Search</button>
```

Comment:
I remembered that the idea was going to be that this was going to do a separate API call, and would lead to the college-info.html page with the school ID, rather than calling loadSchoolInfo here. Is that the plan?